src/__data__/languages.json

```diff
@@ -60,6 +60,20 @@
       }
     ]
   },
+  "pt": {
+    "name": "Portuguese",
+    "nativeName": "Português",
+    "country": "br",
+    "antLocale": "pt_BR",
+    "dayjsLocale": "br",
```

Comment:
Unless I'm mistaken, I think the dayjs locale is `pt-br`?

https://github.com/iamkun/dayjs/blob/dev/src/locale/pt-br.js
https://github.com/iamkun/dayjs/blob/dev/src/locale/br.js